### DISCUSSION TITLE
Navbar dropdown Glitch in Chrome

### DISCUSSION THREAD

**Discussion by aryanshridhar** [encode/django-rest-framework#7798] [Potential Issue]
## Checklist

- [x] I have verified that that issue exists against the `master` branch of Django REST framework.
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] This is not a usage question. (Those should be directed to the [discussion group](https://groups.google.com/forum/#!forum/django-rest-framework) instead.)
- [ ] This cannot be dealt with as a third party library. (We prefer new functionality to be [in the form of third party libraries](https://www.django-rest-framework.org/community/third-party-packages/#about-third-party-packages) where possible.)
- [x] I have reduced the issue to the simplest possible case.
- [ ] I have included a failing test as a pull request. (If you are unable to do so we can still accept the issue.)

## The problem
Navbar dropdown closing as soon as the user tries to toggle it in Chrome .
The expected behaviour was recorded using Mozilla firefox .

## Expected behavior
![peek6O6WH0](https://user-images.githubusercontent.com/53977614/76848569-efaa6d80-6869-11ea-9c50-300af0958ba2.gif)

## Actual behavior

![peekS1SNH0](https://user-images.githubusercontent.com/53977614/76848216-524f3980-6869-11ea-9d0c-2ece84d342e5.gif)

**Comment by lovelydinosaur** [comment]
Is this for chrome on mobile or desktop?
(I wasn't able to replicate on the desktop)

**Comment by aryanshridhar** [comment]
This problem was occurring to me on both the devices .
Desktop Chrome version : Version 80.0.3987.87 (Official Build) (64-bit)
Mobile Chrome version : 80.0.3987.149

When I moved to my Console tab to see what's wrong, I saw these messages lined up as errors:

<pre>jquery-1.8.1-min.js:2 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. </pre>
